src/change-of-program/schema.js

```diff
@@ -0,0 +1,120 @@
+import definitions from '../common/definitions';
+import _ from 'lodash';
+
+module.exports = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'REQUEST FOR CHANGE OF PROGRAM OR PLACE OF TRAINING',
+  type: 'object',
+  additionalProperties: false,
+  definitions: _.pick(definitions, [
+    'fullName',
+    'address',
+    'phone',
+    'ssn',
+    'school',
+    'bankAccount',
+    'serviceBefore1977',
+    'date',
+    'dateRange'
+  ]),
+  properties: {
+    veteranFullName: {
+      $ref: '#/definitions/fullName'
+    },
+    veteranAddress: {
+      $ref: '#/definitions/address'
+    },
+    dayPhone: {
+      $ref: '#/definitions/phone'
+    },
+    nightPhone: {
+      $ref: '#/definitions/phone'
+    },
+    vaFileNumber: {
+      type: 'string'
+    },
+    email: {
+      type: 'string',
+      format: 'email'
+    },
+    veteranSocialSecurityNumber: {
+      $ref: '#/definitions/ssn'
+    },
+    benefit: {
```

Comment:
This form is for modifying your existing benefit. Select one should be accurate.